ReactiveUI/PortableLibraryStubs.cs

```diff
@@ -10,7 +8,16 @@ public class PlatformRegistrations : IWantsToRegisterStuff
     {
         public void Register(Action<Func<object>, Type> registerFunction)
         {
+            if (IsRunningFromNUnit()) return;
```

Comment:
Unfortunately it does not work on my test project. According to Splat source code https://github.com/paulcbetts/splat/blob/master/Splat/ModeDetector.cs there is no way on how to detect a unit test, but my solution works.